### PR TITLE
Tasks: gguf file as a snippet param

### PR DESCRIPTION
**Pull request by krampstudio**

### CODE REVIEW

**Review by coyotte508 — APPROVED** [review]

File: packages/tasks/src/local-apps.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ LLAMA_CURL=1 make
 		`## Load and run the model
 ./main \\
 	--hf-repo "${model.id}" \\
-	-m file.gguf \\
+	-m {{GGUF_FILE}} \\
```
[the remark this answers is not in the document]
Comment (julien-c) [reply packages/tasks/src/local-apps.ts:60]: potentially we could even define here the different possible values (i.e. here `model.ggufFilePaths`) so the selection component could be used by third party libraries too

maybe overkill though, cc @coyotte508

**Review by julien-c — APPROVED** [review]

File: packages/tasks/src/local-apps.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (krampstudio) [reply packages/tasks/src/local-apps.ts:73]: Something like https://github.com/huggingface/huggingface.js/pull/699/commits/5331d0730ab66615e22dc56a8d818cbce6cac4ea ?

[the remark this answers is not in the document]
Comment (julien-c) [reply packages/tasks/src/local-apps.ts:73]: hmmm imo let's not do it for now. And we can reassess later if there's more need for it.

[the remark this answers is not in the document]
Comment (krampstudio) [reply packages/tasks/src/local-apps.ts:60]: ok, reverting

**Pull request merged**